### ISSUE TITLE
Pretty print Juno Config

### ISSUE THREAD

**Issue by IronGauntlets** [NethermindEth/juno#1053]
Config is logged [here](https://github.com/NethermindEth/juno/blob/87fc0ab52271dee53f2ad2efa253fb6e13a1a689/node/node.go#L165) on a single line. For better readability, we should pretty print it on multiple lines

**Comment by joshklop** [comment]
Oh didn't see that you already had a PR up. Nice!

**Comment by wojciechos** [comment]
@IronGauntlets seems like PendingPollInterval is displayed in different format than used to be, e.g for 5s it displays 5000000000:

``` 
root@localhost:~# docker run nethermindeth/juno:v0.5.0-rc0-62-g88f476a --pending-poll-interval 5s
      _
      | |
      | |_   _ _ __   ___
  _   | | | | | '_ \ / _ \
 | |__| | |_| | | | | (_) |
  \____/ \__,_|_| |_|\___/ v0.5.0-rc0-62-g88f476a

Juno is a Go implementation of a Starknet full-node client created by Nethermind.

Running Juno with Config:
{
  "LogLevel": "info",
  "HTTPPort": 6060,
  "DatabasePath": "/var/lib/juno",
  "Network": "mainnet",
  "EthNode": "wss://mainnet.infura.io/ws/v3/81fa89d1cba1494ab1b30f5755fb8989",
  "Pprof": true,
  "Colour": false,
  "PendingPollInterval": 5000000000,
  "Metrics": true,
  "P2P": false,
  "P2PAddr": "",
  "P2PBootPeers": ""
}

**Comment by wojciechos** [comment]
@IronGauntlets seems like PendingPollInterval is displayed in different format than used to be, e.g for 5s it displays 5000000000:

``` 
root@localhost:~# docker run nethermindeth/juno:v0.5.0-rc0-62-g88f476a --pending-poll-interval 5s
      _
      | |
      | |_   _ _ __   ___
  _   | | | | | '_ \ / _ \
 | |__| | |_| | | | | (_) |
  \____/ \__,_|_| |_|\___/ v0.5.0-rc0-62-g88f476a

Juno is a Go implementation of a Starknet full-node client created by Nethermind.

Running Juno with Config:
{
  "LogLevel": "info",
  "HTTPPort": 6060,
  "DatabasePath": "/var/lib/juno",
  "Network": "mainnet",
  "EthNode": "wss://mainnet.infura.io/ws/v3/81fa89d1cba1494ab1b30f5755fb8989",
  "Pprof": true,
  "Colour": false,
  "PendingPollInterval": 5000000000,
  "Metrics": true,
  "P2P": false,
  "P2PAddr": "",
  "P2PBootPeers": ""
}

**Comment by joshklop** [comment]
The `pendingPollInterval` issue is solved by #1061. Would you like to create a separate issue for it @wojciechos?

**Comment by wojciechos** [comment]
> The `pendingPollInterval` issue is solved by #1061. Would you like to create a separate issue for it @wojciechos?

No need for separate issue, glad it's solved!
